ftest/startup/pom.xml

```diff
@@ -50,6 +30,32 @@
   <build>
     <plugins>
       <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-surefire-plugin</artifactId>
+        <configuration>
+          <skipTests>true</skipTests>
+        </configuration>
+      </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-failsafe-plugin</artifactId>
+        <executions>
+          <execution>
+            <goals>
+              <goal>integration-test</goal>
+              <goal>verify</goal>
+            </goals>
+            <configuration>
+              <argLine>-Dlog4j.configuration=${file.uri.prefix}${basedir}/src/test/resources/log4j.xml
+                -Dnuxeo.log.dir=${project.build.directory}
+                -DsomePassword=${some.password}
+                ${it.failsafe.argLine}
+              </argLine>
+            </configuration>
+          </execution>
+        </executions>
+      </plugin>
+      <plugin>
```

Comment:
`maven-surefire-plugin` and `maven-failsafe-plugin` aren't needed since no tests are run